src/renderers/webgl/edge.ts

```diff
@@ -181,6 +189,21 @@ export class EdgeRenderer {
       this.managers.edges.unmount(this.lineSegment)
     }
 
+    const strokesShouldMount = isVisible && this.renderer.zoom > MIN_STROKE_ZOOM
+
+    if (strokesShouldMount && !this.strokes && this.edge.style?.stroke) {
+      this.strokes = new LineStrokes(this.renderer.edgesContainer, this.lineSegment, this.edge.style.stroke).moveTo(this.x0, this.y0)
```

Comment:
ditto, feels like this should be created/torn down in the update method, and only positioned here.